resources/definitions/strateo3d_IDEX420.def.json

```diff
@@ -0,0 +1,132 @@
+            "skirt_brim_minimal_length": { "default_value": 333 },
+            "speed_layer_0": { "value": "20" },
+            "speed_travel_layer_0": { "value": "100" },
+            "speed_prime_tower": { "value": "speed_topbottom" },
+            "speed_print": { "value": "50" },
+            "speed_support": { "value": "speed_wall" },
+            "speed_support_interface": { "value": "speed_topbottom" },
+            "speed_topbottom": { "value": "math.ceil(speed_print * 20/35)" },
+            "speed_travel": { "value": "150" },
+            "speed_wall": { "value": "math.ceil(speed_print * 3/4)" },
+            "speed_wall_0": { "value": "math.ceil(speed_wall * 2/3)" },
+            "speed_wall_x": { "value": "speed_wall" },
+            "support_angle": { "value": "50" },
+            "support_bottom_distance": {"value": "extruderValue(support_bottom_extruder_nr if support_bottom_enable else support_infill_extruder_nr, 'support_z_distance/2') if support_type == 'everywhere' else 0", "maximum_value_warning": "machine_nozzle_size*1.5" },
+            "support_interface_enable": { "default_value": true },
+            "support_interface_height": { "value": "layer_height*3" },
+            "support_interface_offset": { "value": "support_offset" },
+            "support_top_distance": {"value": "extruderValue(support_roof_extruder_nr if support_roof_enable else support_infill_extruder_nr, 'support_z_distance')", "maximum_value_warning": "machine_nozzle_size*1.5" },
+            "support_use_towers": { "default_value": true },
+            "support_xy_distance": { "value": "line_width * 1.7" },
+            "support_xy_distance_overhang": { "value": "wall_line_width_0" },
+            "support_z_distance": { "value": "layer_height*2", "maximum_value_warning": "machine_nozzle_size*1.5" },
+            "switch_extruder_prime_speed": { "value": "retraction_prime_speed" },
+            "switch_extruder_retraction_amount": { "value": "7" },
+            "switch_extruder_retraction_speeds": {"value": "retraction_retract_speed"},
+            "top_bottom_thickness": { "value": "3*layer_height", "minimum_value_warning": "layer_height*2" },
+            "top_thickness": { "value": "top_bottom_thickness" },
+            "top_layers": { "value": "0 if infill_sparse_density == 100 else math.ceil(round(top_thickness / resolveOrValue('layer_height'), 4))"},
+            "bottom_thickness": { "value": "top_bottom_thickness-2*layer_height+layer_height_0" },
+            "bottom_layers": { "value": "999999 if infill_sparse_density == 100 else math.ceil(round(((bottom_thickness-resolveOrValue('layer_height_0')) / resolveOrValue('layer_height'))+1, 4))"},
+            "travel_avoid_distance": { "value": "3 if extruders_enabled_count > 1 else machine_nozzle_tip_outer_diameter / 2 * 1.5" },
+            "wall_thickness": { "value": "wall_line_width_0 + wall_line_width_x" },
+			
+			"acceleration_enabled": { "value": false },
```

Comment:
```suggestion
```
DRY (Don't Repeat Yourself) principle. These values are already default, so no need to reset them here.

Reply:
Hello nallath, this vale was, you are right, equal to the default one. I removed the line from IDEX420 definition files